control/view/lootbox_view_controller.py

```diff
@@ -103,27 +104,34 @@ async def handle_lootbox_claim(
                 )
                 jail_cog: Jail = self.bot.get_cog("Jail")
                 jail_announcement = f"<@{member_id}> delved too deep looking for treasure and got sucked into a wormhole that teleported them into jail."
-                duration = 2 * 60
-                member = interaction.guild.get_member(member_id)
+                duration = random.randint(1*60, 3*60)
+                member = interaction.user
                 success = await jail_cog.jail_user(
                     guild_id, self.bot.user.id, member, duration
                 )
                 if not success:
                     time_now = datetime.datetime.now()
-                    affected_jails = self.database.get_active_jails_by_member(guild_id, member.id)
-                    affected_jail = affected_jails[0]
-                    event = JailEvent(
-                        time_now,
+                    members = self.database.get_active_jails_by_member(guild_id, member.id)
```

Comment:
this function returns jails, not members, so the name should stay as affected_jails